createCard/index.js

```diff
@@ -26,6 +28,17 @@ const { getInputParams, reportError, validateLeankitUrl } = require( "../leankit
 		payload.typeId = typeId;
 	}
 
+    if ( customId ) {
+		payload.customId = customId;
+	}
+    if ( externalLink ) {
+		payload.externalLink = {
+            url: externalLink,
+            label: "Link to Github"
```

Comment:
Maybe we could collect the label too and it could default to "Link to Github" if not provided. Seems like people might use this to link back to things in other systems.

Reply:
I have added a code change to accept a 'linkLabel' as optional (with default of "Link to Github")